### DISCUSSION TITLE
Split korge and korlibs

### DISCUSSION THREAD

**Discussion by soywiz** [korlibs/korge#2099] [Ideas and discussions]
Once everything is in its place again, we can try to split korge and the other libraries. To avoid duplication of the buildSrc folder, we can create a github repository with it, and include it via git submodules.

```
repo: buildSrc
repo: korge
repo: korlibs5
```

Then we can still avoid duplicating all that code, while still being able to split stuff. CI should be faster. I wouldn't split it further to avoid wasting time on that.
Then we can maybe include here in this repo the intellij plugin.

**Comment by itboy87** [comment]
That's a great idea; that's what I'm going to suggest, using Git submodules.
